Controller/CRUDController.php

```diff
@@ -361,11 +361,8 @@ public function editAction($id = null)
         $form->setData($object);
         $form->handleRequest($request);
 
-        if ($this->getRestMethod() == 'POST') {
-            $form->submit($request);
-
+        if ($request->isMethod("POST")) {
```

Comment:
inside $form->isValid() it is call it. https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/Form.php#L726